### ISSUE TITLE
Plugin not working with latest eslint version (8.1.0)

### ISSUE THREAD

**Issue by damianpereira86** [kwoding/eslint-plugin-ui-testing#19]
After updating to eslint latest version (8.1.0) I'm getting this error:

```
❯❯  npx eslint cypress

Oops! Something went wrong! :(

ESLint: 8.1.0

TypeError: Failed to load plugin 'ui-testing' declared in '.eslintrc.json': Class extends value undefined is not a constructor or null
Referenced from: C:\repos\{project_folder}\automation\.eslintrc.json
    at Object.<anonymous> (C:\repos\{project_folder}\automation\node_modules\@typescript-eslint\experimental-utils\dist\ts-eslint\CLIEngine.js:12:34)   
    at Module._compile (C:\repos\{project_folder}\automation\node_modules\v8-compile-cache\v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (C:\repos\p{project_folder}\automation\node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous> (C:\repos\{project_folder}\automation\node_modules\@typescript-eslint\experimental-utils\dist\ts-eslint\index.js:14:14)       
    at Module._compile (C:\repos\{project_folder}\automation\node_modules\v8-compile-cache\v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
```

Going back to the previous version fixes the issue.

**Comment by kwoding** [comment]
Thanks for raising this issue. As far as I can see there are couple of breaking changes in the eslint api. Need to investigate and update the rules.

**Comment by kwoding** [comment]
Just tested and added support for eslint 8, new release published: `eslint-plugin-ui-testing@2.0.0`. Let me know if you still encounter issues.

**Comment by damianpereira86** [comment]
Thanks, I'll try it out!
